Modules/Forum/classes/class.ilForumCronNotification.php

```diff
@@ -231,6 +233,15 @@ public function sendCronForumNotification(ilDBStatement $res, int $notification_
 
             $row['ref_id'] = $ref_id;
 
+            $top_item_ref_id = $this->tree->getParentId($ref_id);
```

Comment:
Have you considered to use $tree->checkForParentType(..., 'crs|grp') here or should only direct child elements be supported?